Challenge-4/Bot_1/src/agent.ts

```diff
@@ -1,35 +1,28 @@
 import {
   Finding,
   HandleTransaction,
-  LogDescription,
   TransactionEvent,
   ethers,
   getEthersProvider,
 } from "forta-agent";
 import {
-  CONFIGURATOR_PROXY,
   ASSET_INFO,
-  LIQUIDATE_CF,
-  BORROW_CF,
-  SET_GOVERNOR,
-  BORROW_PYIR,
-  SUPPLY_KINK,
-  BORROW_KINK,
-  SUPPLY_PYIR,
   USDC_TOKEN_ETH,
   USDC_TOKEN_ARB,
+  RESULTS_ARRAY,
 } from "./constants";
 import { createFinding } from "./findings";
-import { getCollateralAsset, getAddress, getConfigurator } from "./helper";
+import Helper from "./helper";
 import { NetworkManager } from "forta-agent-tools";
 
 let configuratorProxy: string | undefined;
 let networkManager: any;
```

Comment:
```suggestion
let networkManager: NetworkManager;
```
In general, we try to avoid declaring the type of a variable from being `any`. Because that pretty much defeats the purpose of TypeScript. If you make it a specific type, then you'll be certain that it will always be of that type. 🙂